fail to make quick-release (rsync error)

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):BUG REPORT

**Kubernetes version** (use `kubectl version`):Client Version: version.Info{Major:"1", Minor:"6+", GitVersion:"v1.6.0-alpha.0.2850+4398e1edb00b7c-dirty", GitCommit:"4398e1edb00b7cd00b56abb9673467746063d148", GitTreeState:"dirty", BuildDate:"2017-01-06T09:06:04Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"6+", GitVersion:"v1.6.0-alpha.0.2753+f95362f953b681-dirty", GitCommit:"f95362f953b681b495822e99bb772246312c25f9", GitTreeState:"dirty", BuildDate:"2017-01-04T08:46:56Z", GoVersion:"go1.7.4", Compiler:"gc", Platform:"linux/amd64"}

**What happened**:
```
# make quick-release
+++ [0106 16:59:51] Verifying Prerequisites....
+++ [0106 16:59:52] Building Docker image kube-build:build-9901a0c10c-4-v1.7.4-1
+++ [0106 16:59:56] Syncing sources to container
ERROR: password file must be owned by root when running as root
rsync error: syntax or usage error (code 1) at authenticate.c(200) [sender=3.1.1]
Makefile:361: recipe for target 'quick-release' failed
make: *** [quick-release] Error 1
```
**How to reproduce it** (as minimally and precisely as possible):

the owner of file is not right

--- a/build/common.sh
+++ b/build/common.sh
@@ -19,6 +19,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+USER_ID=$(id -u)
+GROUP_ID=$(id -g)
+
 DOCKER_OPTS=${DOCKER_OPTS:-""}
 DOCKER=(docker ${DOCKER_OPTS})
 DOCKER_HOST=${DOCKER_HOST:-""}
@@ -407,6 +410,8 @@ function kube::build::clean() {
 # Set up the context directory for the kube-build image and build it.
 function kube::build::build_image() {
   mkdir -p "${LOCAL_OUTPUT_BUILD_CONTEXT}"
+  # Make sure the context directory owned by the right user for syncing sources to container.
+  chown -R ${USER_ID}.${GROUP_ID} "${LOCAL_OUTPUT_BUILD_CONTEXT}"
 
   cp /etc/localtime "${LOCAL_OUTPUT_BUILD_CONTEXT}/"
 
@@ -494,7 +499,7 @@ function kube::build::ensure_data_container() {
       --name "${KUBE_DATA_CONTAINER_NAME}"
       --hostname "${HOSTNAME}"
       "${KUBE_BUILD_IMAGE}"
-      chown -R $(id -u).$(id -g)
+      chown -R ${USER_ID}.${GROUP_ID}
         "${REMOTE_ROOT}"
         /usr/local/go/pkg/
     )

